docs/readmes/orc8r/upgrade_1_6.md

```diff
@@ -0,0 +1,92 @@
+---
+id: upgrade_1_6
+title: Upgrade to v1.6
+hide_title: true
+---
+
+# Upgrade to v1.6
+
+This guide covers upgrading Orchestrator deployments from v1.5 to v1.6. Upgrades which skip versions (e.g. v1.4 to v1.6) are not explicitly documented or supported at this time.
+
+The v1.6 upgrade follows a standard procedure
+
+- Upgrade Terraform modules
+- Terraform install
+
+## Prerequisites
+
+### Terraform version
+
+This document was tested using Terraform version `0.15.5`. You can use [`tfenv`](https://github.com/tfutils/tfenv) to install a particular Terraform version.
+
+### Terraform state
+
+If you are using local Terraform state (the default), ensure all Terraform state files are located in your working directory before proceeding.
+
+To verify this, `terraform show` should list existing state, rather than outputting `No state`.
+
+### Postgres version
+
+[AWS is terminating support for Postgres 9.6 on November 11, 2021](https://aws.amazon.com/blogs/database/upgrading-from-amazon-rds-for-postgresql-version-9-5/), in accordance with the [Postgres versioning policy](https://www.postgresql.org/support/versioning/). Postgres 9.6 was the default Orc8r DB target for the past several releases.
+
+As of Magma release v1.6, the new default Orc8r DB target is Postgres 12.6. *If you are upgrading from a previous release, you will need to handle this change*.
+
+The recommended path is
+1. Pin the Terraform DB engine to your existing version (likely `9.6.15`)
+2. Separately, at a later date, [use Terraform to upgrade to Postgres 12.6](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance)
```

Comment:
Users cannot directly upgradeable from 9.6 -> 12.6. This is the upgrade path
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.PostgreSQL.html
<img width="1066" alt="Screen Shot 2021-06-30 at 5 02 22 PM" src="https://user-images.githubusercontent.com/8224854/124045642-f14b6680-d9c4-11eb-8d6f-47f3f6171bfa.png">

Reply:
Good info. I view that as implementation detail though, we can figure it out as part of https://github.com/magma/magma/issues/7871